### PR TITLE
add tina's paper

### PR DESCRIPTION
**Pull request by tinatn29**
I added an entry (nanarong;npj24) to db/citations.yml with some paper information

### CODE REVIEW

File: db/citations.yml [hunk 1]
```diff
@@ -8556,6 +8556,46 @@ nakam;jmchema17:
...
   wwwpub: http://slapper.apam.columbia.edu/bib-eu9iifae/papers/nakam_jmca17.pdf
   year: '2017'
+nanarong;npj24:
+  ackno: This work was funded by Toyota Research Institute, grant number PO-002332.
+  author:
+    - Tanaporn Na Narong
+    - Zoe N. Zachko
+    - Steven B. Torrisi
+    - Simon J. L. Billinge
+  doi:
```
Comment (sbillinge) [inline db/citations.yml:8566]: this needs an empty string `''`

File: db/citations.yml [hunk 2]
```diff
@@ -8556,6 +8556,46 @@ nakam;jmchema17:
...
+  entrytype: article
+  facility:
```
Comment (sbillinge) [inline db/citations.yml:8568]: here we list any facilities where data were collecteed.  empty string if none.

File: db/citations.yml [hunk 3]
```diff
@@ -8556,6 +8556,46 @@ nakam;jmchema17:
...
+  grant:
+  journal: Npj Comput. Mater.
+  month: oct
```
Comment (sbillinge) [inline db/citations.yml:8571]: mt string (add this when it appears)

File: db/citations.yml [hunk 4]
```diff
@@ -8556,6 +8556,46 @@ nakam;jmchema17:
...
+  nb: ''
+  note: to be submitted
```
Comment (sbillinge) [inline db/citations.yml:8573]: change this to the arxiv reference when we get that.

File: db/citations.yml [hunk 5]
```diff
@@ -8556,6 +8556,46 @@ nakam;jmchema17:
...
+    enables exploration of information content and can inform experimental design 
+    when choosing between different methods.
+  synopsis: Random forest models trained on XANES, PDF, and both combined 
+    and the feature importance analysis allow us to combine information 
+    from heterogeneous experimental spectra and investigate the information 
+    content and complementarity between the two inputs.
+  tags: ml, xanes, pdf, random-forest
+  title: 'Use of machine learning in experiment design for multi-modal analysis of materials: 
+  x-ray absorption near-edge spectra (XANES) and pair distribution functions (PDF)'
+  url: https://doi.org/
```
Comment (sbillinge) [inline db/citations.yml:8596]: mt string here

**Review by sbillinge — COMMENTED** [review]
pls see inline

**Comment by tinatn29** [comment]
See the new PR (Tina paper 2)-- I added these edits^ and the arxiv link

**Pull request closed without merging**